tools/build_executable.py

```diff
@@ -88,22 +88,31 @@
         shutil.rmtree(unreal_tmp_dir, ignore_errors=True)
 
     if not args.skip_clone_github_repo:
-        
+
         # remove our temporary repository directory to ensure a clean build
         if os.path.exists(repo_dir):
             spear.log(f"Repository exists, removing: {repo_dir}")
             shutil.rmtree(repo_dir, ignore_errors=True)
 
         # clone repo with submodules
-        cmd = ["git", "clone", "--recurse-submodules", "https://github.com/isl-org/spear", repo_dir]
+        cmd = [
```

Comment:
@RachithP I like it better the old way. If one of these `cmd` definitions can fit on one line, we should fit it on one line. If the `cmd` definition requires multiple lines, then I like the formatting in this file, but it shouldn't be mandatory that we put it on multiple lines if it's already so concise.